docs/userGuide/syntax/code.md

```diff
@@ -2,15 +2,22 @@
 
 #### Themes
 
-MarkBind can present formatted code blocks, be it fenced or inline, with either **light** or **dark** themes. The default is dark.
+MarkBind can present formatted code blocks, be it fenced or inline, with either **light** or **dark** themes. The default is dark <trigger for="modal:code-dark-example" trigger="click">_(click for an example)_</trigger>.
+
+<modal header="Dark Code Theme" id="modal:code-dark-example" large >
+
+<div style="overflow: hidden;">
+  <img src="/images/codeDarkTheme.png" width="750" alt="Theme example" />
+</div>
```

Comment:
```suggestion
<div>
  <pic src="annotateSampleImage.png" />
</div>
```

Try replacing the inner img with a `pic` component that markbind supports :)

Reply:
Seems to work as you say, I think this is a better solution than CSS magic.

One note though, on Surface Pro 7 dimensions, pic appears smaller than the div and img version and is a little worse. This is because the div version cuts off the image, but the pic version squeezes it in.

Pic version:
<img width="495" alt="Screenshot 2022-04-19 at 12 14 40 AM" src="https://user-images.githubusercontent.com/59471320/163838334-47be7ae7-e128-403a-9284-cc55a7a1ea2e.png">

Div and img version: 
<img width="495" alt="Screenshot 2022-04-19 at 12 14 40 AM" src="https://user-images.githubusercontent.com/59471320/163838431-788e5176-a2d9-4f22-aff3-201ff7602cd5.png">

Reply:
> Try replacing the inner img with a `pic` component that markbind supports :). I think this might fix the overflow issues. The overflow issue seems to affect the whole page and doesn't seem to be the same issue as what tlylt was saying though.

The overflow issue that @kaixin-hc  is referring to is likely to be #1797, pending @ong6 update :)

Reply:
So overall, content doesn't scale the best with width - but because the overall goal of showing that the colours are different comes across, I think while not ideal it might be okay?

Reply:
I think its better with Pic, since the image is not getting cut off randomly (which might seem weird for users)